backend/gateway/mux/pprof.go

```diff
@@ -0,0 +1,10 @@
+package mux
+
+import (
+	"net/http"
+	"net/http/pprof"
+)
+
+func pprofHandlers(httpMux *http.ServeMux) {
+	httpMux.HandleFunc("/debug/pprof/", pprof.Index)
```

Comment:
let's just fold this inline in mux.go, no need for the indirection to this new file

Reply:
yeah sure